client/src/test/scala/org/http4s/client/middleware/RetrySpec.scala

```diff
@@ -24,17 +24,17 @@ class RetrySpec extends Http4sSpec {
   "Retry Client" should {
     "Retry bad requests" in {
       val max = 2
-      var attempts = 1
-      val policy = (attmpts: Int) => {
+      var attemptsCounter = 1
+      val policy = (attempts: Int) => {
         if (attempts >= max) None
```

Comment:
You'll notice this name does not change since it was referring to the wrong thing before this change. (even though the behavior was correct)